### PR TITLE
Embed the testthat catch code

### PR DESCRIPTION
**Pull request by jimhester**
This removes the hard testthat dependency for isoband, which should
make the ggplot2 stack much lighter overall.

I just put testthat authors as the people, @hadley let me know if we want to do something different.

### CODE REVIEW

**Comment by jimhester** [comment]
Fixes https://github.com/wilkelab/isoband/issues/19

**Comment by clauswilke** [comment]
Thanks. There seems to be a build error on Windows.

I'm fine with however you want to acknowledge the testthat authors. @hadley Any input from your end?

File: DESCRIPTION [hunk 1]
```diff
@@ -11,16 +11,17 @@ Authors@R:
              family = "Pedersen",
              role = "aut",
              email = "thomasp85@gmail.com",
-             comment = c(ORCID = "0000-0002-5147-4711")))
+             comment = c(ORCID = "0000-0002-5147-4711")),
+      person(given = "testthat authors",
```
Comment (hadley) [inline DESCRIPTION:15]: Maybe make it `testthat and Catch` authors?

**Comment by clauswilke** [comment]
Thanks!

**Pull request merged**